Controller/Dashboard.php

```diff
@@ -0,0 +1,51 @@
+<?php declare(strict_types=1);
+
+namespace KaroIO\MessengerMonitorBundle\Controller;
+
+use KaroIO\MessengerMonitorBundle\FailedMessage\FailedMessageRepository;
+use KaroIO\MessengerMonitorBundle\Locator\ReceiverLocator;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Messenger\Transport\Receiver\MessageCountAwareInterface;
+use Twig\Environment;
+
+class Dashboard
```

Comment:
I'd also prefer a `Controller` suffix. I didn't realize at first that a few of these were controllers